scripts/local_testnet/genesis.json

```diff
@@ -0,0 +1,851 @@
+{
+  "config": {
+    "chainId": 4242,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
+    "mergeForkBlock": 0,
+    "shanghaiTime": 0,
+    "terminalTotalDifficulty": 0
+  },
+  "alloc": {
+    "0x0000000000000000000000000000000000000000": {
```

Comment:
probably a question for later / 4844 testnet - is it worth pre-allocating some funds to a known account for testing blob txs?

Reply:
Ohh good point. I added prefunded accounts with static private keys from https://github.com/sigp/lighthouse/blob/stable/testing/execution_engine_integration/src/execution_engine.rs#L12-L15